### ISSUE TITLE
Surface identity by connection in the CLI

### ISSUE THREAD

**Issue by grampelberg** [linkerd/linkerd2#2605]
### What problem are you trying to solve?

The TLS % in `stat` is confusing and potentially misleading. As it shows aggregates, it is difficult to understand what connections between client/server pairs are encrypted, which aren't and why. Debugging why a connection is not being upgraded to TLS can be frustrating.

### How should the problem be solved?

First, the `tls` column should be removed. Second, introduce a new `identity` command:

```bash
$ linkerd -n emojivoto identity po
 SRC                                     DST                      CLIENT   SERVER   REASON
 web-56955474f7-hsqnn                    emoji-596866d567-6wl6s   web      emoji
 gke-foobar-default-pool-5f6c4d44-7jjh   web-56955474f7-hsqnn                       no_authority_in_http_request
 emoji-596866d567-6wl6s                  172.217.5.110                              not_provided_by_remote
 emoji-596866d567-6wl6s                  mongo-1                                    tcp
$ linkerd -n booksapp identity deploy
 SRC                  DST       CLIENT    SERVER    REASON
 traffic              webapp    default   default
 webapp               books     default   default
 webapp               authors   default   default
 books                authors   default   default
 linkerd-prometheus   webapp                        no_authority_in_http_request
 linkerd-prometheus   books                         no_authority_in_http_request
 linkerd-prometheus   authors                       no_authority_in_http_request
$ linkerd -n linkerd identity deploy
 SRC                                     DST                  CLIENT               SERVER               REASON
 linkerd-web                             linkerd-controller   linkerd-web          linkerd-controller
 linkerd-web                             linkerd-grafana      linkerd-web          linkerd-grafana
 linkerd-controller                      linkerd-prometheus   linkerd-controller   linkerd-prometheus
 linkerd-grafana                         linkerd-prometheus   linkerd-grafana      linkerd-prometheus
 linkerd-prometheus                      linkerd-identity     linkerd-prometheus   linkerd-identity
 gke-foobar-default-pool-5f6c4d44-7jjh   linkerd-web                                                    not_provided_by_remote
 gke-foobar-default-pool-5f6c4d44-7jjh   linkerd-controller                                             not_provided_by_remote
 gke-foobar-default-pool-5f6c4d44-7jjh   linkerd-prometheus                                             not_provided_by_remote
 gke-foobar-default-pool-5f6c4d44-7jjh   linkerd-grafana                                                not_provided_by_remote
 gke-foobar-default-pool-5f6c4d44-7jjh   linkerd-identity                                               not_provided_by_remote
```
#### Open questions

- I've not added namespaces and alternative resource types to this. Any good ideas on how that should show up? Could add columns, but they'll be for both SRC and DST.
- liveness/readiness probes come from the kubelet (node) and show up as an IP address. It would be nice to show that these are liveness/readiness probes and flag that as these will never been TLS'd.
- On my GKE cluster, service traffic of type `LoadBalancer` also appears to originate from the node. Is there a way to label that traffic as cluster external?
- I showed one example of `tcp`. A lot of incoming/outgoing connections rely on HTTPS and won't be seen normally. Can we still show this level of data from metrics?

**Comment by scottcarol** [comment]
More detailed feature requests are now in #2708 and #2709

**Comment by scottcarol** [comment]
Fixed by #2808
